helix-term/src/ui/editor.rs

```diff
@@ -770,6 +770,67 @@ impl Component for EditorView {
 
                 EventResult::Consumed(callback)
             }
+            Event::Mouse(MouseEvent {
+                kind: MouseEventKind::Down(MouseButton::Left),
+                row,
+                column,
+                ..
+            }) => {
+                let editor = &mut cx.editor;
+                let tree = &mut editor.tree;
+                let mut id: Option<ViewId> = None;
+                let mut result = EventResult::Ignored;
+
+                for (view, view_id) in tree.views_with_keys_mut() {
+                    if !view.verify_screen_coords(row as usize, column as usize) {
+                        continue;
+                    }
+
+                    id = Some(view_id);
+
+                    let doc = &mut editor.documents[view.doc];
+                    let jump = (doc.id(), doc.selection(view.id).clone());
+                    view.jumps.push(jump);
+
+                    let pos = view.pos_at_screen_coords(doc, row as usize, column as usize);
+                    // Not in current view. Impossible, but do not panic
+                    if pos == None {
+                        break;
+                    }
```

Comment:
I think you can simply unwrap here since it's guaranteed by `verify_screen_coords` to be valid.

Reply:
Well, true, thanks